CuratedContent/ResourcesforOSPO.md

```diff
@@ -0,0 +1,36 @@
+# Resources for Open Source Program Offices
+
+<!--deck text start-->
+Are you starting an Open Source Program Office (OSPO) or want to ensure the success of an existing OSPO or an open source project? If so, checkout these resources to help.
+<!--deck text end-->
+
+#### Contributed by [Patricia Grubel](http://github.com/pagrubel)
+
+#### Publication date: March 28, 2023
+
+Resource information | Details
+:--- | :---
+Organization type  | Open Source Program Offices
+Websites  |  Github OSPO  - [Github site to enable success on github for new OSPOS](https://github.com/github/github-ospo)<br> OSPO Alliance - [Community Driven Open Source Good Governance](https://ospo-alliance.org/)<br>TODO Group - [Talk Openly Develop Openly, an open community for sharing knowledge for successful OSPOs](https://todogroup.org/)<br> Open Source Guides - [Guides for Open Source Projects](https://opensource.guide/)
+Focus | Success for Open Source Program Offices (OSPOs)
+
+In some organizations the governance and guidance for open source projects fall under the guize of an Open Source Program Office.
```

Comment:
```suggestion
In some organizations the governance and guidance for open source projects falls under the purview of an Open Source Program Office.
```

Reply:
governance and guidance fall - so not accepting this change